swan-cern/values.yaml

```diff
@@ -258,154 +258,61 @@ hadoopTokenGenerator:
-        enable_ruby
-        <record>
-            metrics.${record['_metric_key_']}.user ${record['_metric_user_']}
-            metrics.${record['_metric_key_']}.context ${record['_metric_context_']}
-            metrics.${record['_metric_key_']}.value ${record['_metric_value_']}
-
-            # the below field is needed to run queries that group values by the last segment of the metric context (segments split by '.')
-            # which looks like "metrics.{record['_metric_key_']}_lastelementofmetric_context = {record['metric_value']}"
-            # for example, for a record with below fields
-            #     "data.metrics.spawn.context = LCG_xxx.some_spark_cluster.exception_class"
-            #     "data.metrics.spawn.value = None"
-            # it emits a field "data.metrics.spawn_exception_class = None
-            metrics.${record['_metric_key_']}_${record['_metric_context_'].gsub('-','_').split('.')[-1]} ${record['_metric_value_']}
-        </record>
-        remove_keys ["_metric_user_", "_metric_key_", "_metric_context_", "_metric_value_"]
-    </filter>
-
-    # Emit general log records with type "logs"
-    # Accessible in OpenSearch with pattern monit_private_swan_logs_logs
-    <filter {logs}>
-        @type record_transformer
-        <record>
-            type "logs"
-        </record>
-    </filter>
-
-    # Emit metric log records with type "metrics"
-    # Accessible in OpenSearch with pattern monit_private_swan_logs_metrics
-    <filter {metrics}>
-        @type record_transformer
-        <record>
-            type "metrics"
-        </record>
-    </filter>
-
-    # Set MONIT producer for all logs
-    # Keep raw log data in "raw" field
-    <filter {logs,metrics}>
-        @type record_transformer
-        enable_ruby
-        <record>
-            timestamp ${(time.to_f * 1000).to_i}
-            producer "#{ENV['OUTPUT_PRODUCER']}"
-            raw ${record["log"]} # field named raw is indexed by MONIT/OpenSearch
-        </record>
-        remove_keys ["log"]
-    </filter>
-
-    # Push logs to MONIT endpoint
-    <match {logs,metrics}>
-        @type http
-        endpoint_url    "#{ENV['OUTPUT_ENDPOINT']}"
-        serializer      json
-        http_method     post
-    </match>
-
-    # Forward errors in this pipeline to fluentd pod stdout for debugging
-    <label @ERROR>
-        <match **>
-            @type stdout
-        </match>
-    </label>
+    cacert: /etc/ssl/ca-bundle.crt
+  podSecurityPolicy:
+    enabled: false
+  ## Prometheus Monitoring
+  metrics:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: prometheus-operator
+  resources:
+    requests:
+      cpu: 5m
+      memory: 700Mi
+  configMapConfigs:
+    - fluentd-prometheus-conf
+    - fluentd-sources-conf
+    - fluentd-filters-conf
+    - fluentd-outputs-conf
+  fileConfigs:
+    01_sources.conf: ""
+    02_filters.conf: ""
+    03_dispatch.conf: ""
+    04_outputs.conf: ""
```

Comment:
Do we also need these to an empty string, to prevent fluentd from taking the settings of the upstream chart? Isn't it enough to define configMapConfigs?

Reply:
Unfortunately, we need these to override the default fluentd configuration that is present in the upstream chart.